packages/zowe-explorer-api/.eslintignore

```diff
@@ -1,3 +1,4 @@
+__mocks__/**
```

Comment:
Lines 1 and 4 look like duplicates

Reply:
Addressed in 18f1972ed92a4328ea933a6579ba15efe2665da6